File: 2520-count-the-digits-that-divide-a-number/2520-count-the-digits-that-divide-a-number.py
class Solution(object):
    def countDigits(self, num):
        """
        :type num: int
        :rtype: int
        """
        temp = num
        count = 0

        while temp > 0:
            t = temp % 10
            if num % t == 0:
                count += 1
            temp /= 10
        
        return count

File: 2520-count-the-digits-that-divide-a-number/2520-count-the-digits-that-divide-a-number.java
class Solution {
    public int countDigits(int num) {
        int temp = num;
        int count = 0;

        while(temp > 0){
            int t = temp % 10;
            if(num % t == 0){
                count++;
            }
            temp /= 10;
        }
        return count;
    }
}